client/src/components/navbar-component/Dropdown.jsx

```diff
@@ -0,0 +1,22 @@
+import PropTypes from "prop-types";
+import "./Dropdown.css";
+import DropdownItem from "./DropdownItem";
+
+function Dropdown({ isAdmin }) {
+  return (
+    <div className="dropdown-container">
+      <div className="dropdown-title">Profile</div>
+      <div className="dropdown-items">
```

Comment:
<img width="1787" alt="image" src="https://github.com/uoftblueprint/tces/assets/58835213/4ccd62a5-ea0e-4bad-bbc5-0cbca7fb4b92">
Currently this is where the dropdown is opening on my end